.github/workflows/job-postcommit-java-nexmark-direct.yml

```diff
@@ -0,0 +1,201 @@
+              --tempLocation=$TEMPLOCATION 
+              --influxDatabase=$INFLUXDATABASE 
+              --influxHost=$INFLUXHOST 
+              --baseInfluxMeasurement=$BASEINFLUXMEASUREMENT 
+              --exportSummaryToInfluxDB=$EXPORTSUMMARYTOINFLUXDB 
+              --influxRetentionPolicy=$INFLUXRETENTIONPOLICY 
+              --queryLanguage=sql 
+              --streaming=true 
+              --suite=SMOKE 
+              --manageResources=false 
+              --monitorJobs=true 
+              --enforceEncodability=true 
+              --enforceImmutability=true "
+      - name: Run *** RUN NEXMARK IN ZETASQL BATCH MODE USING DIRECT RUNNER ***
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: >
+            :sdks:java:testing:nexmark:run -Pnexmark.runner=:runners:direct-java 
+            -Pnexmark.args=" 
+              --runner=DirectRunner 
+              --bigQueryTable=$BIGQUERYTABLE 
+              --bigQueryDataset=$BIGQUERYDATASET 
+              --project=$PROJECT 
+              --resourceNameMode=$RESOURCENAMEMODE 
+              --exportSummaryToBigQuery=$EXPORTSUMMARYTOBIGQUERY 
+              --tempLocation=$TEMPLOCATION 
+              --influxDatabase=$INFLUXDATABASE 
+              --influxHost=$INFLUXHOST 
+              --baseInfluxMeasurement=$BASEINFLUXMEASUREMENT 
+              --exportSummaryToInfluxDB=$EXPORTSUMMARYTOINFLUXDB 
+              --influxRetentionPolicy=$INFLUXRETENTIONPOLICY 
+              --queryLanguage=zetasql 
+              --streaming=false 
+              --suite=SMOKE 
+              --manageResources=false 
+              --monitorJobs=true 
+              --enforceEncodability=true 
+              --enforceImmutability=true "
+      - name:   RUN *** NEXMARK IN ZETASQL STREAMING MODE USING DIRECT RUNNER ***
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: >
+            :sdks:java:testing:nexmark:run -Pnexmark.runner=:runners:direct-java 
+            -Pnexmark.args=" 
+              --runner=DirectRunner 
+              --bigQueryTable=$BIGQUERYTABLE 
+              --bigQueryDataset=$BIGQUERYDATASET 
+              --project=$PROJECT 
+              --resourceNameMode=$RESOURCENAMEMODE 
+              --exportSummaryToBigQuery=$EXPORTSUMMARYTOBIGQUERY 
+              --tempLocation=$TEMPLOCATION 
+              --influxDatabase=$INFLUXDATABASE 
+              --influxHost=$INFLUXHOST 
+              --baseInfluxMeasurement=$BASEINFLUXMEASUREMENT 
+              --exportSummaryToInfluxDB=$EXPORTSUMMARYTOINFLUXDB 
+              --influxRetentionPolicy=$INFLUXRETENTIONPOLICY 
+              --queryLanguage=sql 
+              --streaming=true 
+              --suite=SMOKE 
+              --manageResources=false 
+              --monitorJobs=true 
+              --enforceEncodability=true 
+              --enforceImmutability=true "
```

Comment:
Is the only difference here the streaming variable and the queryLanguage? If so, can we just make these separate jobs and use a matrix to run with each variable type?

Reply:
Same comment applies below in the other workflows. In fact, if the runner is the only difference, we could probably just consolidate to a single workflow and use matrix builds on the runner.

Reply:
Working in the changes.